drivers/soundwire/bus.c

```diff
@@ -1276,7 +1276,7 @@ int sdw_configure_dpn_intr(struct sdw_slave *slave,
 	return ret;
 }
 
-static int sdw_slave_set_frequency(struct sdw_slave *slave)
+int sdw_slave_set_frequency(struct sdw_slave *slave)
```

Comment:
Can you reword the commit message, it is hard to follow.

Expose the sdw_slave_set_frequency() to allow re-programming the peripheral frequency to follow the curr_dr_freq, which can change after the stream is prepared.

Or something.